lsp-mode.el

```diff
@@ -1188,6 +1188,19 @@ Deprecated. Use `lsp-repeatable-vector' instead. "
   "Display lsp error message with FORMAT with ARGS."
   (message "%s :: %s" (propertize "LSP" 'face 'error) (apply #'format format args)))
 
+(defun lsp--fatal-error (format &rest args)
+  "Show an lsp error message using `error'.
+FORMAT is the `format' format to use, while ARGS provides the
+fields to it."
+  (error "%s :: %s" (propertize "LSP" 'face 'error)
+         (apply #'format format args)))
+
+(defun lsp--user-error (format &rest args)
```

Comment:
I don't see any effect from this propertize - the message is displayed with default font.

Reply:
Yes, that is the issue currently.